### DISCUSSION TITLE
Filter Data from relational Table

### DISCUSSION THREAD

**Discussion by DevrayePatil** [Budibase/budibase#7655] [Help]
I have created order Table and Items details table. I want to display items from Order

**Comment by andz-bb** [comment]
Hey @DevrayePatil, could you provide a few more details about what you're looking to do? Is it a Budibase DB you're using or an external DB? Are you looking to filter a table based on a selection from another related table? Thanks

**Reply by DevrayePatil** [reply]
Yes, I am using Budibase DB in which I have used two tables Item details and Order. In these tables has relation one order have have many Item details and I want to filter which order has selected that order related item details I want to show
